utils/v2tofhirr4/helpers.bal

```diff
@@ -45,7 +45,7 @@ function checkComputableAntlr(ANTLR[] antlrList) returns boolean {
 }
 
 // Comparison Operaions
-function isContain(string x, string[] valueList) returns boolean {
+isolated function isContain(string x, string[] valueList) returns boolean {
```

Comment:
You can use langlib function `indexOf` or `some` without implement your own function

Reply:
Fixed with 4e64c6bbe86866f77e8b9c6c727389888e77d90d